library/modules/EventManager.cpp

```diff
@@ -902,6 +935,26 @@ static void manageEquipmentEvent(color_ostream& out) {
             equipment.push_back(item);
         }
     }
+
+    // now handle events
+    std::for_each(equipment_pickups.begin(), equipment_drops.end(), [&](InventoryChangeData& data) {
```

Comment:
equipment_pickups != equipment_drops. 

Is this intentional?

Reply:
no, it was not